test/js/bun/stream/direct-readable-stream-suspense.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { Suspense } from "react";
+import { renderToReadableStream } from "react-dom/server";
+
+if (!import.meta.resolveSync("react-dom/server").endsWith("server.bun.js")) {
```

Comment:
How is this test run?

Reply:
Amended the test so that it uses bun:test.